src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/CreditCardMapper.php

```diff
@@ -83,8 +83,8 @@ protected function getUrlToComputop($merchantId, $data, $length)
             ComputopApiConfig::DATA => $data,
             ComputopApiConfig::LENGTH => $length,
             ComputopApiConfig::URL_BACK => $this->getAbsoluteUrl(
-                    $this->application->path($this->config->getCallbackFailureRedirectPath())
-                )
+                $this->application->path($this->config->getCallbackFailureRedirectPath())
```

Comment:
I would also move this parameter inside function, especially if it's used only once in the class